config.json.template

```diff
@@ -0,0 +1,36 @@
+{
+    "irods_client": {
+        "host": "localhost",
+        "port": 1247,
+
+        "zone": "tempZone",
+
+        "rodsadmin": {
+            "username": "rods",
+            "password": "rods"
+        }
+    },
+
+    "plugins": {
+        "static_bucket_resolver": {
+            "name": "static_bucket_resolver",
+            "mappings": {
+                "test-bucket": "/tempZone/home/rods"
+            }
+        },
+
+        "static_authentication_resolver": {
+            "name": "static_authentication_resolver",
+            "users": {
+                "s3_rods": {
+                    "username": "rods",
+                    "secret_key": "s3rodspass"
```

Comment:
want to continue to use alice, to show it's not a rodsadmin, but we can tweak later.

Reply:
done